docs/commands.md

```diff
@@ -133,12 +133,12 @@ WHERE actor.age >= director.age AND actor.age > 32
 It is also possible to specify equality predicates within nodes using the curly braces as such:
 
 ```sh
-(:president {name:"Barack Obama"})-[:won]->(:state)
+(:president {name:"Jed Bartlett"})-[:won]->(:state)
```

Comment:
I think it's better to relate to a fictional President (from The West Wing) than a real President.